External/FEXCore/include/FEXCore/Core/CoreState.h

```diff
@@ -49,6 +111,12 @@ namespace FEXCore::Core {
     uint16_t FCW;
     uint16_t FTW;
 
+    uint32_t _pad2[1];
+    // Reference counter for FEX's per-thread deferred signals.
```

Comment:
Calling it a reference counter is technically correct, but it's not clear what exactly is being counted. Maybe something like `Reference counter for FEX's per-thread deferred signals. Counts the nesting depth of program sections that cause signals to be deferred`?

Reply:
Resolved.